### PR TITLE
Define strategy how to transfer HTML attributes to the component and accordingly modify the `TabsItem` component

### PR DESCRIPTION
**Pull request by mbohal**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -164,6 +164,16 @@ To keep React UI consistent and predictable the following guidelines should be o
 
 1. If component accepts the `children` prop it should be either required or the element
    should return `null` when no children are provided.
+2. When forwarding HTML attributes to the component the following rules should
```
Comment (mbohal) [inline CONTRIBUTING.md:167]: We still need to deal with `Modal` and `Radio` components. See issue: https://github.com/react-ui-org/react-ui/issues/496

**Review by dacerondrej — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -164,6 +164,16 @@ To keep React UI consistent and predictable the following guidelines should be o
...
+    be observed:
+    1. If the component internally instantiates an interactive
+        (clickable/editable) element, the attributes should be forwarded to it.
+    2. If the component internally instantiates multiple interactive
+        (clickable/editable) elements, the attributes should be forwarded to
+        them.
```
Comment (adamkudrna) [inline CONTRIBUTING.md:173]:
```suggestion
        (clickable/editable) elements, the attributes should be forwarded to
        all of them.
```

?

Comment (adamkudrna) [inline CONTRIBUTING.md:173]: Maybe these two could be merged:

```suggestion
    1. If the component internally instantiates one or more interactive
        (clickable/editable) elements, the attributes should be forwarded to 
       all of them.
```

**Review by adamkudrna — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (mbohal) [reply CONTRIBUTING.md]: done

Comment (mbohal) [reply CONTRIBUTING.md]: done

**Review by adamkudrna — APPROVED** [review]
Cool!

**Review by bedrich-schindler — APPROVED** [review]

**Pull request merged**